### ISSUE TITLE
Stuck `discover-proc-kubelet-cmdline` crashes `longhorn-driver-deployer`

### ISSUE THREAD

**Issue by jaredkipe** [longhorn/longhorn#815]
Crashing in `longhorn-driver-deployer` with logs

```
time="2019-10-10T22:07:42Z" level=debug msg="User specified the driver csi"
10/10/2019 3:07:42 PM time="2019-10-10T22:07:42Z" level=debug msg="Deploying CSI driver"
10/10/2019 3:07:42 PM time="2019-10-10T22:07:42Z" level=error msg="failed to get arg root-dir. Need to specify \"--kubelet-root-dir\" in your Longhorn deployment yaml.: failed to get cmdline of proc kubelet: failed to deploy proc cmdline detection pod discover-proc-kubelet-cmdline: pods \"discover-proc-kubelet-cmdline\" already exists"
10/10/2019 3:07:42 PM time="2019-10-10T22:07:42Z" level=fatal msg="Error deploying driver: failed to get arg root-dir. Need to specify \"--kubelet-root-dir\" in your Longhorn deployment yaml.: failed to get cmdline of proc kubelet: failed to deploy proc cmdline detection pod discover-proc-kubelet-cmdline: pods \"discover-proc-kubelet-cmdline\" already exists"
```

There is a 'completed' pod named `discover-proc-kubelet-cmdline` with the logs.

```
Proc found: kubelet
9/7/2019 2:03:56 PM kubelet--address=0.0.0.0--streaming-connection-idle-timeout=30m--v=2--pod-infra-container-image=rancher/pause:3.1--authorization-mode=Webhook--event-qps=0--cluster-domain=cluster.local--tls-cipher-suites=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305--cgroups-per-qos=True--hostname-override=dev1-ubu16-01--allow-privileged=true--anonymous-auth=false--cni-conf-dir=/etc/cni/net.d--network-plugin=cni--enforce-node-allocatable=--volume-plugin-dir=/var/lib/kubelet/volumeplugins--authentication-token-webhook=true--make-iptables-util-chains=true--fail-swap-on=false--root-dir=/var/lib/kubelet--kubeconfig=/etc/kubernetes/ssl/kubecfg-kube-node.yaml--cni-bin-dir=/opt/cni/bin--read-only-port=0--resolv-conf=/etc/resolv.conf--client-ca-file=/etc/kubernetes/ssl/kube-ca.pem--cloud-provider=--cluster-dns=10.43.0.10--cgroup-driver=cgroupfs
```

Deleting the `discover-proc-kubelet-cmdline` pod by hand, and scaling down/up the `longhorn-driver-deployer` deployment resolved the issue.

**Comment by shuo-wu** [comment]
Which Longhorn version are you using? 
This bug should have been fixed by [this commit](https://github.com/longhorn/longhorn-manager/commit/df0c1fab73e8974f2f801750308a6603980cab2a)

**Comment by yasker** [comment]
Yeah, talked with @jaredkipe I think it happened in v0.5.0 (please correct me if I am wrong)

So v0.6.x should already contain the fix.

**Comment by yasker** [comment]
Close as fixed in v0.6.x. Feel free to reopen if it's not the case.
